inc/template-functions.php

```diff
@@ -166,7 +166,7 @@ function nightingale_custom_page_colour( $classes ) {
     $page_header_colour = get_theme_mod( 'page_header_colour', '' );
 
     if(!empty($page_header_colour)){
-        $page_header_colour_name = 'page-header-color--' . $colour_array[ $page_header_colour ];
+        $page_header_colour_name = 'page-header-colour--' . $colour_array[ $page_header_colour ];
```

Comment:
if we changing color to colour we also need to do it in the sass. (page-colours.scss)

Reply:
Changed. Hopefully I didn't miss any of them. I've left the American spelling in the main colours.scss for now as Malcolm is working on that ticket.

Reply:
I missed one, however now fixed. Grep is saying there are no more :|